builds/jenkins-build-pack-lambda/Jenkinsfile

```diff
@@ -752,28 +767,99 @@ def createSubscriptionFilters(stackName) {
 
   try {
     filter_json = sh(
-      script: "aws logs describe-subscription-filters --output json --log-group-name \"${lambda}\" --region ${region}",
+      script: "aws logs describe-subscription-filters --output json --log-group-name \"${lambda}\" --region ${service_config.region}",
```

Comment:
logs are in the deployment account. We ought to be executing the aws commands with deployment account credentials. Do you agree?

Reply:
updated the changes in describe, delete and put